README.md

```diff
@@ -0,0 +1,32 @@
+# Anyscale Starter Templates
+
+These templates are a set of minimal examples & tutorial for customers to run on the anyscale platform.
+
+## Contributing Guide
+
+If the template is generic to Ray & Ray libraries, please consider adding the template in: https://github.com/ray-project/ray/tree/master/doc/source/templates
+
+
+To add a template:
+
+1. Add your template as a directory under `templates/<your-template-name>`
+
+    For example:
+
+    ```text
+    templates/my-awesome-template
+        <name-of-your-template>/
+            README.md
+            <name-of-your-template>.ipynb/py
+    ```
+
+    Your template does not need to be a Jupyter notebook. It can also be presented as a
+    Python script.
```

Comment:
Can this be mix of both?

For example: https://github.com/ray-project/ray-educational-materials/tree/main/NLP_workloads/Text_generation

Reply:
yes it can, if you plan to add a cell at the end of the notebook to deploy as a job or as a service, you have to convert the notebook into python or include a python version since anyscale job & services don't support notebooks.